nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-ldap/src/main/java/org/nuxeo/ecm/directory/ldap/LDAPServerDescriptor.java

```diff
@@ -115,12 +116,7 @@ public String getBindDn() {
 
     @XNode("bindPassword")
     public void setBindPassword(String bindPassword) {
-        if (bindPassword == null) {
-            // no password means empty pasword
-            this.bindPassword = "";
-        } else {
-            this.bindPassword = bindPassword;
-        }
+        this.bindPassword = Objects.requireNonNullElse(bindPassword, ""); // no password means empty password
```

Comment:
You can't just do a `this.bindPassword = bindPassword != null ? bindPassword : "";`

Reply:
I'm not a big fan of ternary expression, especially when we can use an elvis operator in another language. `requireNonNullElse` allows to not repeat `bindPassword` which I prefer.